xml/osc_building.xml

```diff
@@ -196,5 +195,192 @@ The buildroot was: /var/tmp/build-root/openSUSE_Tumbleweed-x86_64</screen>
   <remark>toms 2017-08-18: integrate content from  obs_build_preinstall.xml;
   using a _preinstallimage file</remark>
  </sect1>
- 
+
+ <sect1 xml:id="sec.obs.building.appimage">
+  <title>As &obs.ai;</title>
+  <para>
+   To create an &obs.ai;, you need:
```

Comment:
hm, this seems like it is the same list as obs_package_formats.xml, more or less. can you create a small XML file with just that list in it? otherwise, we have to maintain it twice.

Reply:
I know, it seems these two sections grow organically. Not sure what's the best solution here. On the one hand, it would be nice to have one dedicated chapter/section which just contains the requisites. But on the other hand, everything in one place would be better...
I need to think about it more or maybe we could discuss this in person.